source/_integrations/otbr.markdown

```diff
@@ -12,6 +12,17 @@ ha_config_flow: true
 ---
 
 The Open Thread Border Router integration allows calling an Open Thread Border Router's REST API from Python and via WebSocket.
-The integration is automatically set up when the "Silicon Labs Multiprotocol" add-on is installed.
+
+This integration is installed automatically when the **Open Thread Border Router** add-on is installed. This happens when you do one of the following procedures:
+
+- [Enabling Thread on Home Assistant Yellow](https://yellow.home-assistant.io/procedures/enable-thread/)
+- [Enabling Thread on Home Assistant SkyConnect](https://skyconnect.home-assistant.io/procedures/enable-thread/)
+
+The integration is installed automatically when the **Silicon Labs Multiprotocol** add-on is installed. This happens when you do one of the following procedures:
+
+- [Enabling Multiprotocol on Home Assistant Yellow](https://yellow.home-assistant.io/guides/enable-multiprotocol/)
+- [Enabling Multiprotocol on Home Assistant SkyConnect](https://skyconnect.home-assistant.io/procedures/enable-multiprotocol/)
```

Comment:
Since the dedicated Thread is the promoted solution for now, let's drop the links here.

```suggestion

The integration is also installed automatically when enabling the experimental **Silicon Labs Multiprotocol** support.
```